src/server/session.ts

```diff
@@ -693,6 +720,10 @@ module ts.server {
         }
 
         onMessage(message: string) {
+            if (this.logger.isVerbose()) {

```

Comment:
Can we add a 'logIfNecessary' helper.  Tihs pattern is repeated in several places it seems.